source/jormungandr/tests/check_utils.py

```diff
@@ -497,6 +501,8 @@ def is_valid_journey(journey, tester, query):
         g = s.get('geojson')
         g is None or shape(g)
 
+        assert (s['from']['id'] == previous_destination_uri)
+        previous_destination_uri = s['to']['id']
```

Comment:
it's tested at L496 https://github.com/CanalTP/navitia/pull/1915/files#diff-75dec1e2d23a2e9bc7163d480c7808edR496